src/auth/screens/events.screen.js

```diff
@@ -559,7 +560,7 @@ class Events extends Component {
                 item.type === 'PullRequestReviewCommentEvent') &&
                 <View style={styles.subtitleContainer}>
                   <Text numberOfLines={3} style={styles.subtitle}>
-                    {item.payload.comment.body.replace(/(\r\n|\n|\r)/gm, ' ')}
+                    {emojifyText(item.payload.comment.body.replace(/(\r\n|\n|\r)/gm, ' '))}
```

Comment:
Do you mind moving `/(\r\n|\n|\r)/gm` into it's own constant so it's easier for new comers to know what is this doing?